registrations/permissions.py

```diff
@@ -125,8 +130,11 @@ def has_object_permission(
 class CanAccessSignupGroup(CanAccessSignup):
     @staticmethod
     def _has_object_update_permission(request: Request, obj: SignUpGroup) -> bool:
-        if request.user.is_superuser or request.user.is_registration_admin_of(
-            obj.publisher
+        if (
+            request.user.is_superuser
+            or request.user.is_registration_admin_of(obj.publisher)
+            or request.user.is_admin_of(obj.publisher)
+            and obj.registration.created_by_id == request.user.id
```

Comment:
I would use parentheses here as well:
```
if (
            request.user.is_superuser
            or request.user.is_registration_admin_of(obj.publisher)
            or (
                request.user.is_admin_of(obj.publisher)
                and obj.registration.created_by_id == request.user.id
            )
        )
```

Reply:
Based on https://docs.python.org/3/reference/expressions.html#operator-precedence parentheses shouldn't be required, but I guess they improve readability so might be a nice addition.

Reply:
Yes, I think it's a nice readability improvement. Changed as suggested now.